bootloader/lk.go

```diff
@@ -361,9 +361,9 @@ func (l *lk) ExtractRecoveryKernelAssets(recoverySystemDir string, sn snap.Place
 		// error case, we cannot be extracting a recovery kernel and also be
 		// called with !opts.PrepareImageTime (yet)
 
-		// TODO:UC20: however this codepath will likely be exercised when we
-		//            support creating new recovery systems from runtime
-		return fmt.Errorf("internal error: ExtractRecoveryKernelAssets not yet implemented for a runtime lk bootloader")
+		// TODO:UC20: this codepath is exercised when creating new
```

Comment:
yeah to implement this we need some new bits in lkenv, it's not insurmountable, it's very similar to existing code we have but it can wait a bit until we need it